core/arch/arm/kernel/ldelf_loader.c

```diff
@@ -85,7 +85,7 @@ TEE_Result ldelf_load_ldelf(struct user_mode_ctx *uctx)
 
 	prot = TEE_MATTR_URX;
 	if (IS_ENABLED(CFG_CORE_BTI))
-		prot |= TEE_MATTR_GAURDED;
+		prot |= TEE_MATTR_GUARDED;
```

Comment:
This fix belongs to the commit that introduced the typo

Reply:
Shifted